pkg/virt-controller/watch/replicaset/OWNERS

```diff
@@ -0,0 +1,7 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+reviewers:
+  - sig-compute-reviewers
+approvers:
+  - sig-compute-approvers
+labels:
+  - area/replicaset-controller
```

Comment:
Do we have these labels? @dhiller @brianmcarey ?

Reply:
We don't :) I misunderstood how it works

Reply:
Please see https://github.com/kubevirt/project-infra/blob/main/docs/labels.md

Also there is docs on how to add these if required - however I'd think that the SIG involved would decide what labels we want for kubevirt

Reply:
I see i.e. `area/controller` https://github.com/kubevirt/project-infra/blob/main/docs/labels.md#area/controller